### PR TITLE
Improve site formatting

### PR DESCRIPTION
**Pull request by nirs**
Improve site formatting to make it easier to maintain and have nicer output.

Preview:
- https://github.com/nirs/cobra/blob/site-formatting/site/content/user_guide.md
- https://github.com/nirs/cobra/blob/site-formatting/site/content/active_help.md

### CODE REVIEW

File: site/content/user_guide.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ While you are welcome to provide your own organization, typically a Cobra-based
...
-        commands.go
-        here.go
-      main.go
+▾ appName/
+  ▾ cmd/
+      add.go
+      your.go
+      commands.go
+      here.go
+  main.go
```
Comment (nirs) [inline site/content/user_guide.md:13]: Not sure if main.go is indented correctly. We can replace it with tree output which is much more clear to users and also used later below.

```console
appName
├── cmd
│   ├── add.go
│   ├── your.go
│   ├── commands.go
│   └── here.go
└── main.go
```

**Review by marckhouzam — APPROVED** [review]
Nice!
Thanks!

File: site/content/user_guide.md (hunk 1 above)
Comment (marckhouzam) [reply site/content/user_guide.md:13]: Yeah, feel free to change this if you prefer.

Comment (marckhouzam) [reply site/content/user_guide.md:13]: In a follow-up 😄

**Pull request merged**